### ISSUE TITLE
Tab order is not accessible from keyboard.

### ISSUE THREAD

**Issue by malin-boyanov** [microsoft/fluentui#21008]
### Environment Information

- **Package version(s)**: Fluent UI React 7.177.2
- **Browser and OS versions**: Microsoft Edge Version 96.0.1054.53 (Official build) (64-bit)

### Describe the issue:

Menu items are not accessible using keyboard tab button. Only the active menu item has a "tabindex" different from "-1". Those "tabindex" property is set dynamically and can't be set by adding "tabindex" in parameters of "CommandBar" component.

#### Actual behavior:

Using "Tab" button on page skip all menu items but the active one and continue with the next element of the page beneath menu.

#### Expected behavior:

Tab button select every menu item.

**Comment by layershifter** [comment]
Can you please fill the issue template fully and attach the repro case? 🙏
